cg/cli/post_process/utils.py

```diff
@@ -21,3 +32,36 @@ def get_post_processing_service_from_run_name(
             f"Run name {run_name} does not match with any known sequencing run name pattern"
         ) from error
     return getattr(context.post_processing_services, device)
+
+
+def get_unprocessed_runs_info(context: CGConfig, instrument: str) -> list[UnprocessedRunInfo]:
+    """Return a list of unprocessed runs for a given instrument or for all instruments."""
+    runs: list[UnprocessedRunInfo] = []
+    possible_instruments: list[str] = ["pacbio"]  # Add more instruments here
+    instruments_to_process = [instrument] if instrument != "all" else possible_instruments
```

Comment:
Can this be extracted into a smaller function? Like `instruments_to_check`?

Reply:
done